.golangci.yml

```diff
@@ -82,6 +82,7 @@ linters-settings:
           - "**/*/{bridges,instrumentation}/**/*.go"
           - "!**/*/bridges/prometheus/*.go" # prometheus bridge is meant to use the SDK
           - "!**/*/instrumentation/runtime/*.go" # runtime instrumentation is meant to use the SDK
+          - "!**/*/instrumentation/github.com/gin-gonic/gin/otelgin/*.go" # gin instrumentation is meant to use the SDK
```

Comment:
No, gin instrumentation is not meant to use the SDK.
There is a `test` subfolder which is a separate module. You can put tests that need the SDK in there, they will be ignored by this linter automatically.

Reply:
This has been removed before, you can refresh to view the latest code.